sdk/digitaltwins/Azure.DigitalTwins.Core/src/Azure.DigitalTwins.Core.csproj

```diff
@@ -13,7 +13,7 @@
   <PropertyGroup>
     <PackageTags>IoT;DigitalTwins;$(PackageCommonTags)</PackageTags>
     <Description>SDK for the Azure Digital Twins service</Description>
-    <Version>1.1.0-beta.1</Version>
+    <Version>1.0.1</Version>
     <ApiCompatVersion>1.0.0</ApiCompatVersion>
```

Comment:
Apparently the way this works is we update it to the last GA'd version. Since this is our first time shipping after GA, it remains 1.0.0, but after this ships and we update version numbers, it be 1.0.1.

<https://github.com/Azure/azure-sdk-for-net/blob/master/CONTRIBUTING.md#api-compatibility-verification>